### PR TITLE
fix table header showing dummy scrollbar

### PR DESCRIPTION
**Pull request by mayank99**
## Changes

Fixes https://github.com/iTwin/iTwinUI/issues/1814#issuecomment-1921876655

After some experimenting, I found that I was able to remove the scrollbar track and spin buttons (while preserving alignment) by setting `scrollbar-color` to transparent.

## Testing

Tested in Chrome (and Edge) 121 on Windows. Other platforms unaffected.

| Before | After |
| --- | --- |
| image | image |

## Docs

Added changeset.

### CODE REVIEW

File: .changeset/hip-wasps-accept.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': patch
+'@itwin/itwinui-css': patch
+---
+
+Fixed an issue where `Table` was showing a dummy scrollbar track inside the table header in Chrome 121.
```
Comment (rohan-kadkol) [inline .changeset/hip-wasps-accept.md:6]: Should we mention Chrome 121 *+ Windows*?

```suggestion
Fixed an issue where `Table` was showing a dummy vertical scrollbar track inside the table header in Chrome 121 + Windows.
```

Comment (rohan-kadkol) [reply .changeset/hip-wasps-accept.md:6]: Actually, this also happened to me on Edge 121 + Windows.

**Review by rohan-kadkol — COMMENTED** [review]
> Tested in Chrome 121 on Windows. Other platforms unaffected.

I was able to reproduce this on Chrome 121 + Windows and Edge 121 + Windows.

File: .changeset/hip-wasps-accept.md (hunk 1 above)
Comment (mayank99) [reply .changeset/hip-wasps-accept.md]: Edge is just Chromium under the hood, so not necessary, but mentioning Windows makes sense 👍  [`13f29d6`](https://github.com/iTwin/iTwinUI/pull/1816/commits/13f29d6fcbbd9407f06aba6ef3f28122f51c94fe)

**Review by rohan-kadkol — APPROVED** [review]

**Comment by mayank99** [comment]
> I was able to reproduce this on Chrome 121 + Windows and Edge 121 + Windows.

Yes, Edge also uses Chromium, which added support for standard scrollbar properties in v121. I verified it's fixed there too.

| Before | After |
| --- | --- |
| image | image |

**Pull request merged**